### PR TITLE
Move pixels to iframe

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What is the purpose of this pull request?
Move the pixel apps to inside an iframe, for security reasons.

#### What problem is this solving?
The pixel apps (which can, and _will_ be written by third-party developers) could access all the information the user has on the parent `window` object, which implies some security concerns. This moves the pixel apps to be enclosed inside an `iframe`, without access to the main parent window.

#### How should this be manually tested?
[Workspace](https://pixelframe--storecomponents.myvtex.com/eletrodomesticos/d).

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [x] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: node/index.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import pixelFrame from './handlers/pixelFrame'
+import Resources from './resources'
+
+const prepare = (handler: (ctx: StoreContext) => Promise<void>) => async (ctx: StoreContext) => {
+  ctx.resources = new Resources(ctx)
+
+  try {
+    await handler(ctx)
+  } catch (err) {
+    console.error('err', err)
```
Comment (luciannojunior) [inline node/index.ts]: Remove it?

File: react/StoreWrapper.js [hunk 2]
```diff
@@ -102,7 +103,7 @@ class StoreWrapper extends Component {
         <IconPack />
         <PixelProvider>
           <DataLayerProvider value={{ dataLayer: window.dataLayer }}>
-            <ExtensionPoint id="store/pixel" />
+            <PixelManager />
             <ExtensionPoint id="store/rc" />
```
Comment (brunojdo) [inline react/StoreWrapper.js]: Does this will exist?

File: store/blocks.json [hunk 3]
```diff
@@ -10,7 +10,8 @@
       "header.full",
       "product-details",
       "footer"
-    ]
+    ],
+    "context": "ProductContext"
```
Comment (brunojdo) [inline store/blocks.json]: This is already defined on the` interface.json`

File: store/routes.json [hunk 4]
```diff
@@ -23,12 +23,12 @@
   },
   "store.search#department": {
     "path": "/:department/d",
-    "canonical": "/:department"
+    "canonical": "/:brand"
```
Comment (brunojdo) [inline store/routes.json]: poop merge

Comment (lucasecdb) [reply store/routes.json]: :hankey:

File: react/StoreWrapper.js (hunk 2 above)
Comment (lucasecdb) [reply react/StoreWrapper.js]: no, it won't

**Pull request merged**